ui/app/components/usage/page/index.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@ember/component';
+import { service } from '@ember/service';
+import type ApiService from 'vault/services/api';
+import type { getUsageDataFunction, UsageDashboardData } from '@hashicorp/vault-reporting/types/index';
+import { allEngines } from 'vault/helpers/mountable-secret-engines';
+import { allMethods } from 'vault/helpers/mountable-auth-methods';
+import type FlagsService from 'vault/services/flags';
+export default class ClientsActivityComponent extends Component {
+  @service declare readonly api: ApiService;
+  @service declare readonly flags: FlagsService;
+
+  handleFetchUsageData: getUsageDataFunction = async () => {
+    //TODO: Update client with typed response after the API is updated https://hashicorp.atlassian.net/browse/VAULT-35108
+    const response = await this.api.sys.systemReadUtilizationReport();
+    const data = response.data as UsageDashboardData;
+    // Replace engine names with display names if available
+    allEngines().forEach((engine) => {
+      if (engine.type in data.secret_engines) {
```

Comment:
I noticed in the screenshot that "KV" is just one secret engine in the graph - is there a reason we don't differentiate between KV v1 and v2? It seems like that'd be useful. 

I'm also not sure using the `displayName` from the mountable helpers is the most accurate here. It's a little weird to see `Username & Password` in the chart since lots of auth methods accept those parameters and `userpass` feels like a more accurate descriptor when it comes to data visualizations. 

Leaving as is and just returning `engine.type` is probably clearer or defining a separate mapping here might make more sense.

Reply:
Resolving as this will be decided as part of the aforementioned design review.